tee_supplicant/include/tee_supplicant.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2023 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef _TEE_SUPPLICANT_H
+#define _TEE_SUPPLICANT_H
+
+void tee_supp_mutex_lock(struct k_mutex *mu);
```

Comment:
This functions will be used outside of `tee_supplicant.c`?

Reply:
Probably. I don't know yet.

Reply:
If so, I think we should leave them static.

Reply:
Ok